include/odp/api/spec/ipsec.h

```diff
@@ -1226,14 +1226,21 @@ typedef struct odp_ipsec_status_t {
  * e.g. RFC 4302 and 4303). Resulting packets are well formed, reconstructed
  * original IP packets, with IPSEC headers removed and valid header field values
  * restored. The amount and content of packet data before the IP header is
- * undefined.
+ * undefined. Some amount of TFC padding may follow the IP packet payload,
+ * in which case packet length is larger than protocol headers indicate.
+ * In tunnel mode, a TFC dummy packet have both IPv4 and IPv6 flags cleared, and
+ * the packet may contain some (dummy) data. Furthermore, inline IPSEC
+ * processing may drop dummy packets.
```

Comment:
It seems @bala-manoharan and me would still prefer separate is_dummy flag. We don't even have to limit it to TFC dummy packets, but might use it later for all kinds of dummy packets.

Reply:
@lumag Would exposing he next header byte also be acceptable? That would similarly seem to be future-proof and would also avoid having to introduce an `odp_packet_has_dummy()` predicate function.

Reply:
In fact I'm more biased towards `odp_packet_get_l3_proto()`/`odp_packet_get_l4_proto()` functions.